### PR TITLE
[ADD] fleet: new configurations doc

### PR DESCRIPTION
**Pull request by larm-odoo**
Adding new fleet documentation- this one is all the configurations in the config column.

### CODE REVIEW

File: content/applications/hr/fleet/configuration.rst
Comment (jcs-odoo) [inline content/applications/hr/fleet/configuration.rst:3]: What about "configuration" only? it is in the "Fleet" category. But see also my suggestion in the main comment.

**Review by jcs-odoo — COMMENTED** [review]
Hi @larm-odoo 

If it fits the structure you have planned for this app's documentation, could you maybe move the configuration info to the app-level page (hr/fleet.rst)? Just a suggestion :)

Cheers :)

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:3]: Hi @jcs-odoo - I have two documents currently written in github for fleet, and I am working on a third. So ultimately, they will all be nested under 'fleet', so I didn't want to make the default fleet doc be this one. 

I had done the format you suggested for other apps, where main doc I wrote covered 90% of things, but this time I was trying to chop the docs up into smaller digestible pieces, so that's why I tried to make separate docs for different items (addling a  vehicle, configurations, reports, repairs, etc).  If you don't think that's appropriate or a good idea, I can combine all the docs into one, but I was hoping to try and make the docs less overwhelming by having them separated out. Does that sound good?

I also know it means that once one doc is approved and posted, I will need to change the TOC so it looks correct with the newly added docs. I didn't want to wait to have the first doc be posted for review, have it go live, then add the rest, one at a time. That is why there are 2 (with another one on the way) posted now for review.  Sound good?

**Review by jero-odoo — CHANGES_REQUESTED** [review]
@larm-odoo Nice work, just a few things to edit.

File: content/applications/hr/fleet/configuration.rst
Comment (jcs-odoo) [reply content/applications/hr/fleet/configuration.rst]: Hi @larm-odoo 
You're right, it's better to have several pull requests rather than a massive one, at least in this documentation repository. Once the first PR is merged, you'll probably have a conflict on the other ones until you fix the toc tree. So that's fine :)

I agree this content only wouldn't make that much sense for the main page. Now that I see it again, I think my suggestion isn't better. "Configuration" alone is too vague (a bit like "misc." or "advanced"). Maybe "vehicle models and manufacturers"? That would be more representative. It's up to you :) 

Have a good day

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:36]: If you look in Runbot, one of the manufacturers is for a bicycle, not a car. Then, if you make a DB using 15.0 form scratch, it is just the 66 manufacturers. So the test data in runbot is adding a bicycle which is not the default. This took me a while to figure out in the beginning since I had added in that bike manufacturer at first, but it's misleading in Runbot.

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:139]: Great- added!

**Review by jero-odoo — APPROVED** [review]

**Comment by larm-odoo** [comment]
Just FYI- this is done in 15.2 runbot. When requesting a new vehicle, in this doc, the word is CREATE but in 15,0 it is NEW- just an FYI so you know. That's really the only difference in the versions for this doc, so I wanted to point that out.

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:21]:
```suggestion
To help ensure no contracts lapse or so no vehicles are left without contracted drivers, Odoo has an
alert in place to let the person responsible for the vehicle contract, know when a contract is about
to expire. Enter the number of days before a contract is going to expire, so that an alert email will be
sent.
```

This paragraph needs to be reworded. Last sentence doesn't make sense and first line is a bit confusing

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:23]: Do you cover this somewhere else?

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:40]: This is confusing.... the page says: 
`New Vehicle Request
An employee cannot request a new car if there are more than 
1,000
 existing cars availables.`
```suggestion
Set a limit on how many vehicles are in the fleet by setting a limit on requests. When using the
```

Doesn't say limit the fleet size-- but only deter folks from requesting new cars

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:41]: where does salary configuration come into play here? Where is this?

Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:41]: change amount to number

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:58]: example? you talk numbers twice here and its a little confusing.

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:69]: name is missing gui label

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:75]:
```suggestion
Many manufacturers have a variety of models that they produce. When adding a vehicle to the fleet, it
```

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:76]:
```suggestion
is important to specify the vehicle model being added. Odoo comes with pre-configured models for four
```

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:78]: This sentence doesn't make sense-- if they come preconfigured why do I have to add them?

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:133]: This makes me want to know why this is? Why is it integrated into payroll?

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:152]: This is covered in the other fleet doc? is this the same info?

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:162]: any other European countries?

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:164]: this should be put in an admonition block

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:166]: again salary configuration is familiar to you, but I have no idea where this comes from

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:167]: where is it depreciated?

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:173]: these are monthly figures and should be stated as such in bold in the doc

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:194]: This is a repeat of the other doc. need to differentiate the two I am confused--- which populates which, one can be removed?

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:182]: again over life per year? average?

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [reply content/applications/hr/fleet/configuration.rst:162]: let's test this out or ask an expert-- talking about Belgian based companies so much in your docs makes the software seem like it is just suited for that market, we are worldwide-- and can accommodate lots of different situations. Also this is best suited for a note block or something like that--- important

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:208]: where do I go to see a list of all my vendors in fleet?

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:222]: I would like to see examples

File: content/applications/hr/fleet/configuration.rst
Comment (tiku-odoo) [inline content/applications/hr/fleet/configuration.rst:231]: why should I order them?

**Review by tiku-odoo — COMMENTED** [review]
@larm-odoo 

Good start...

I think this doc needs some more work though. I saw spelling errors after your team looked it over. I also have some suggestions. You repeat a section of this doc in the other doc (https://github.com/odoo/documentation/pull/5759) -- we should never be inputting info twice-- is it housed differently? can we prioritize and create an anchor?

The style seems cookie cut-- I would like to see better flow in these docs. Also keep in mind to write smart for 16/17. (save for example--- "if applicable, click save")

I would like to take a look at this doc again when you draft more changes.

Let me know if you have any questions. 

Thanks,
Tim :+1:

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:41]: The option to request a new vehicle is only seen on the salary configurator. This is the only place an employee can request a new vehicle. I checked with Quentin in Belgium on a lot of these items since I wasn't sure about a lot of  these.

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:40]: True- I guess I saw this as a way to limit the size of the fleet, since people would not be able to request new cars if X old cars are already available - but I can see how that is a leap.

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:58]: I changed "for" to "from" and the manufacturers, so I think it's clearer now. 

Odoo comes with forty five pre-configured models from four major auto manufacturers: Audi, BMW, Mercedes, and Opel.

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:75]: It never occurred to me that there could be a manufacturer with ONE single model =D but I guess it could happen!

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:78]: I updated this so it's more clear- because there's only models configured for 4 manufacturers-most people (outside of Belgium) will need to add models.

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:133]: I added some more text here- vehicles are part of compensation/benefits/etc, so that's why there are parts of fleet that are firmly connected to Payroll.

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:222]: the last image is the model list- is this what you mean? The examples are bicycle, passenger car, minivan, passenger van, etc. It's just below.

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:152]: This is in the new vehicle doc, yes. I noted on there that I'll let @StraubCreative decide if I should remove that section form that doc, or remove it form here and link to that doc, or keep them in both,

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:162]: I did- I asked Quentin abut all of these fields because there was NO way I was able to figure that out myself! =D 

There are unfortunately a bunch of fields that are Belgian-specific or only appear in a Belgian localization. I notice that is the case for a bunch of areas in our HR docs.

Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:162]: I checked again- and the section doesn't even appear for non-Belgian companies So I updated this with an intro sentence.

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:194]: These are actually different forms- one is when adding a new vehicle, this is when adding a new model in general. It is really confusing because 80% of the form is identical- BUT the form name is actually different, so it is not the same. It just looks a lot like the other!

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:208]: There isn't a separate list of vehicle vendors- they are the same vendors that you would use in the purchase app. So when you click ADD you see all the vendors in the database, from Azure Interior to BMW of Buffalo. I added a sentence to clarify this- since there is no dedicated vehicle vendor list.

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:173]: Added this to the top admonition block.

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:182]: it's i grams per kilometer- so it's an average but it's per kilometer (distace) not yearly or monthly (time based). I added it to the second entry to make it more clear.

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:2]: OK, Sam recommends Configuration menu, so I am going with that =)

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:23]: Not in this doc- but I believe I also explain who the "Responsible" person is in the other doc as well.

**Comment by larm-odoo** [comment]
Hi @tiku-odoo - I went through and addressed all your comments. I think everything should be clarified, but if not, let me know!
Ready for a second look.

**Review by tiku-odoo — COMMENTED** [review]
@larm-odoo 

Looks much better! A couple of comments. It looks good. 

See comment: https://github.com/odoo/documentation/pull/5759#pullrequestreview-1674931749

I will approve once you can clarify the duplicate info. I know it's present on both forms; just confused why it's input twice.

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:42]: More than- I added an example block to make this more clear- as well as updated the text here.

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:245]: Yes- this isn't a default order, etc. This is just listing the possible models. I just like having things ordered, so if/where you can rearrange things, I tell people they can. I am the crazy person who would order these by seating capacity =D

**Comment by larm-odoo** [comment]
Hi @tiku-odoo - I addressed everything and added an anchor in this doc (which will be linked to the other doc, the new vehicle doc). This doc needs to be published first so that the following doc can reference this one. Right now the other doc has a code error because it is linking to an anchor in this doc, which doesn't exist yet online. Please check/approve this one first, so I can have this published, then I will get the other doc published. Any issues with this let me know- thank you!

**Comment by larm-odoo** [comment]
When this PR closes, get [[ADD] fleet: add a new vehicle](https://github.com/odoo/documentation/pull/5759#top)
#5759 merged next!

File: content/applications/hr/fleet/configuration.rst
Comment (samueljlieber) [inline content/applications/hr/fleet/configuration.rst:53]:
```suggestion
.. _fleet/configuration/manufacturers:

```

File: content/applications/hr/fleet/configuration.rst
Comment (samueljlieber) [inline content/applications/hr/fleet/configuration.rst:76]:
```suggestion
.. _fleet/configuration/models:

```

File: content/applications/hr/fleet/configuration.rst
Comment (samueljlieber) [inline content/applications/hr/fleet/configuration.rst:84]: Added Eddy Merckx
```suggestion
it is important to specify the vehicle model being added. Odoo comes with pre-configured models five
major auto manufacturers: Audi, BMW, Eddy Merckx, Mercedes, and Opel. If a vehicle *other* than the
pre-configured models from these manufacturers are part of a fleet, the model (and/or manufacturer)
will need to be added to the database.
```

File: content/applications/hr/fleet/configuration.rst
Comment (samueljlieber) [inline content/applications/hr/fleet/configuration.rst:119]: Added Eddy Merckx
```suggestion
+-------+--------------+-------------+-----------+---------------+
| AUDI  | BMW          | Eddy Merckx | Mercedes  | Opel          |
+=======+==============+=============+===========+===============+
| A1    | Serie 1      | SanRemo76   | Class A   | Agilia        |
+-------+--------------+-------------+-----------+---------------+
| A3    | Serie 3      |             | Class B   | Ampera        |
+-------+--------------+-------------+-----------+---------------+
| A4    | Serie 5      |             | Class C   | Antara        |
+-------+--------------+-------------+-----------+---------------+
| A5    | Serie 6      |             | Class CL  | Astra         |
+-------+--------------+-------------+-----------+---------------+
| A6    | Serie 7      |             | Class CLS | AstraGTC      |
+-------+--------------+-------------+-----------+---------------+
| A7    | Serie Hybrid |             | Class E   | Combo Tour    |
+-------+--------------+-------------+-----------+---------------+
| A8    | Serie M      |             | Class GL  | Corsa         |
+-------+--------------+-------------+-----------+---------------+
| Q3    | Serie X      |             | Class GLK | Insignia      |
+-------+--------------+-------------+-----------+---------------+
| Q5    | Serie Z4     |             | Class M   | Meriva        |
+-------+--------------+-------------+-----------+---------------+
| Q7    |              |             | Class R   | Mokka         |
+-------+--------------+-------------+-----------+---------------+
| TT    |              |             | Class S   | Zafira        |
+-------+--------------+-------------+-----------+---------------+
|       |              |             | Class SLK | Zafira Tourer |
+-------+--------------+-------------+-----------+---------------+
|       |              |             | Class SLS |               |
+-------+--------------+-------------+-----------+---------------+

```

File: content/applications/hr/fleet/configuration.rst
Comment (samueljlieber) [inline content/applications/hr/fleet/configuration.rst:135]: Early line break
```suggestion
  not configured, type in the manufacturer and then click :guilabel:`Create` or :guilabel:`Create &
  Edit`.
```

File: content/applications/hr/fleet/configuration.rst
Comment (samueljlieber) [inline content/applications/hr/fleet/configuration.rst:142]: Early line break
```suggestion
  menu. To create a new category, type in the category and then click :guilabel:`Create (new
  category)`.
```

File: content/applications/hr/fleet/configuration.rst
Comment (samueljlieber) [inline content/applications/hr/fleet/configuration.rst:185]: Double closing parenthesis 
```suggestion
- :guilabel:`Total Cost (Depreciated)`: This value is the :guilabel:`Cost (Depreciated)` and the
```

File: content/applications/hr/fleet/configuration.rst
Comment (samueljlieber) [inline content/applications/hr/fleet/configuration.rst:230]:
```suggestion
.. _fleet/configuration/categories:

```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @larm-odoo! Awesome job with the content in this PR, very thorough 👍

I do have a few suggestions after my technical review, which you can review below. 

Please tag me for another look once my suggestions have been addressed, thank you!

File: content/applications/hr/fleet/configuration.rst
Comment (larm-odoo) [reply content/applications/hr/fleet/configuration.rst:84]: Eddy Merckx is actually a bicycle manufacturer, and the pre-loaded model is a bike, not a car. I am updating this to show that.

**Comment by larm-odoo** [comment]
Thank you Sam for your help and review! This is ready for another look.

File: content/applications/hr/fleet/configuration.rst
Comment (samueljlieber) [inline content/applications/hr/fleet/configuration.rst:52]: Extra line 🙂
```suggestion
```

**Comment by samueljlieber** [comment]
Hi @StraubCreative this PR is good to go 👍

**Review by StraubCreative — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**
